cmd/gen-go-sample/schema_v1p2/schema.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schema_v1p2
+
+type SampleConfig struct {
+	Type    string
+	Version string `yaml: config_schema_version`
+	Samples []Sample
+}
+
+type Sample struct {
+	ID              string `yaml: id`
+	Title           string
```

Comment:
Not an issue, just curious: where does the title come from? I don't see anything about a title in the samplegen user guide.

Reply:
Hmmmm I think we found that we needed this after sharing out the user guide. I'll add it there. (It's used the generate this [`sample-metadata`](https://github.com/googleapis/gapic-generator/blob/a591ce660d04b0e9c6876b9836d8aac0e9f65b6d/src/test/java/com/google/api/codegen/gapic/testdata/java/java_library.baseline#L3302-L3305) part, but I don't think anyone needs the `sample-metadata` yet)

Reply:
And fyi, there has been some discussion of moving the metadata to the manifest, but nothing's been decided in that regard.